### PR TITLE
Add Club Penguin Rewritten

### PR DESCRIPTION
**Pull request by BartBoys3**

### CODE REVIEW

File: Club Penguin Rewritten/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      "id": "322892916093747200"
+    },
+    "service": "Club Penguin Rewritten",
+    "description": {
+        "en" : "Waddle around and meet new friends!"
+    },
+    "url": "play.cprewritten.net",
+    "version": "1.0",
+    "logo": "https://bartboys3.github.io/link-shorten/m/cp/cprlogo.jpg",
+    "thumbnail": "https://bartboys3.github.io/link-shorten/m/cp/cprthumb.png",
```
Comment (Bas950) [inline Club Penguin Rewritten/dist/metadata.json]: Upload to images to imgur.

File: Club Penguin Rewritten/presence.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+var elapsed = Math.floor(Date.now()/1000);
+
+presence.on("UpdateData", async () => {
+
+    let data: presenceData = {
+        largeImageKey: "cprlogo"
+    };
+    presence.setActivity(data);
+});
```
Comment (Bas950) [inline Club Penguin Rewritten/presence.ts]: Try this:

```
var presence = new Presence({
    clientId: "619984959247220750",
    mediaKeys: false
});

var elapsed = Math.floor(Date.now()/1000);

presence.on("UpdateData", async () => {
    if (document.URL.includes("/#/login")) {
        let data: presenceData = {
            largeImageKey: "cprlogo",
            details: "In-Game" //Or logging in
        };
        presence.setActivity(data);
    } else {
        presence.setActivity();
    }
});	
```

Comment (Bas950) [reply Club Penguin Rewritten/presence.ts]: It might give an outcome that you like better.

Comment (Bas950) [reply Club Penguin Rewritten/presence.ts]: Also upload proof of it working if you do change it.

**Comment by BartBoys3** [comment]
Images
image
image
image

**Pull request merged**